src/mkrelease.sh

```diff
@@ -1,132 +0,0 @@
-#!/bin/bash
-
-###############################################################################
-#
-#  ./mkrelease.sh [version]
-#        - Creates a release branch and updates necessary files for creating a
-#          release named [version] of the agent
-#
-###############################################################################
```

Comment:
Maybe we should keep this around for a bit just in case?

Reply:
it's in git if we need it.